packages/editor/src/components/collapsible-block-toolbar/style.scss

```diff
@@ -28,21 +28,23 @@
 		width: $border-width;
 		height: $grid-unit-30;
 		background-color: $gray-300;
-		margin-left: $grid-unit;
+		margin-right: $grid-unit - $border-width;
 	}
 
 	// Modified group borders.
 	.components-toolbar-group,
 	.components-toolbar {
 		border-right: none;
+		position: relative;
 
 		&::after {
 			content: "";
 			width: $border-width;
 			height: $grid-unit-30;
 			background-color: $gray-300;
-			margin-top: $grid-unit-05;
-			margin-left: $grid-unit;
+			top: $grid-unit-05;
+			position: absolute;
+			right: - $border-width;
```

Comment:
In some cases we do `-$border-width`, here there's a space between. No strong opinion on which path is better, but might be good to be consistent?